Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'sinatra', require: 'sinatra/base'
+gem 'sinatra-contrib', require: 'sinatra/namespace'
```

Comment:
Needed to add this gem to be able to namespace in Sinatra

Reply:
oo cool

Reply:
looking good